simulator/src/main/java/com/hazelcast/simulator/test/PropertyBinding.java

```diff
@@ -55,16 +51,21 @@
  * <li>Metronome instance in fields annotated with {@link InjectMetronome}</li>
  * <li>Probe instance in fields annotated with {@link InjectProbe}</li>
  * </ol>
- *
+ * <p>
  * The {@link PropertyBinding} also keeps track of all used properties. This makes it possible to detect if there are any unused
  * properties (so properties which are not bound). See {@link #ensureNoUnusedProperties()}.
  */
 @SuppressWarnings("checkstyle:visibilitymodifier")
 public class PropertyBinding {
 
     // properties
-    public int metronomeIntervalUs;
+    public int intervalUs;
```

Comment:
What are the default values here? For me it's looks like a `SleepingMetronome` with `intervalUS = 0` and `ratePerSecond = 0`. Shouldn't this thrown an `IllegalArgumentException` when `MetronomeBuilder::withRatePerSecond(ratePerSecond)` is called in `loadMetronomeClass()`?